README.md

```diff
@@ -30,3 +30,31 @@ You might want to add the new ```.build_ano``` build output folder to your .giti
 
 
 <img src="docs/Toscanini.png" alt="Arturo Toscanini" height="150" width="147"/>
+
+## Getting Started
+
+There is an excellent [getting started guide for ino](http://inotool.org/quickstart) that contains
+everything you should need to get the Blink program running on your board.
+
+## Troubleshooting
+
+HALP!  I am getting:
```

Comment:
ah, no. levity is fine. Just fix the backtick formatting and I'll merge. Thanks.